devito/operator/operator.py

```diff
@@ -219,6 +219,12 @@ def _build(cls, expressions, **kwargs):
 
         # Required for the jit-compilation
         op._compiler = kwargs['compiler']
+
+        # Add any metadata from the profiler
```

Comment:
nitpicking, I wouldn't call it "metadata"

Reply:
hope is better